api/python/quilt3/formats.py

```diff
@@ -312,6 +312,43 @@ def for_meta(cls, meta):
         name = cls._get_name_from_meta(meta)
         return cls.for_format(name)
 
+    @classmethod
+    def show_all_supported_formats(cls):
```

Comment:
Seems like this would be a very useful function to have return a dict, list or other parseable format (i.e. instead of simply printing).

Reply:
Fixed